File: Plug-Ins/Template Editor/ElementDVDR.m
#import "ElementDVDR.h"

@implementation ElementDVDR

- (instancetype)initForType:(NSString *)t withLabel:(NSString *)l
{
    if (self = [super initForType:t withLabel:l]) {
        self.label = l; // Use entire label without separating tooltip
        self.rowHeight = [l componentsSeparatedByString:@"\n"].count * 17;
    }
    return self;
}

- (void)configureGroupView:(NSTableCellView *)view
{
    view.textField.stringValue = self.displayLabel;
}

@end

File: Plug-Ins/Template Editor/Elements/ElementDVDR.swift
import Cocoa

class ElementDVDR: Element, GroupElement {
    required init(type: String, label: String, tooltip: String? = nil) {
        super.init(type: type, label: label, tooltip: "")
        if label.isEmpty {
            self.rowHeight = 1
        } else {
            self.rowHeight = Double(label.components(separatedBy: "\n").count) * 17 + 1
        }
    }
    
    func configureGroup(view: NSTableCellView) {
        view.textField?.stringValue = self.label
    }
}
